server/package.json

```diff
@@ -1,17 +1,31 @@
 {
   "scripts": {
-    "dev": "ts-node-dev --respawn --transpile-only src/index.ts"
+    "dev": "nodemon -x ts-node src/index.ts",
+    "lint": "eslint --ext ts ./src",
+    "lint:fix": "yarn lint --fix",
+    "prettier": "prettier --ignore-path .gitignore --write src"
   },
   "dependencies": {
-    "@types/express": "^4.17.21",
-    "@types/node": "^20.11.24",
     "cors": "^2.8.5",
     "express": "^4.18.3",
     "pg": "^8.11.3",
-    "typescript": "^5.3.3"
+    "reflect-metadata": "^0.2.2",
+    "sequelize": "6.37.2",
+    "sequelize-typescript": "2.1.6",
+    "typescript": "^5.3.3",
+    "winston": "^3.13.0"
   },
   "devDependencies": {
+    "@types/cors": "^2.8.17",
+    "@types/express": "^4.17.21",
+    "@types/node": "^20.11.24",
+    "@typescript-eslint/eslint-plugin": "^6.7.2",
+    "@typescript-eslint/parser": "^6.7.2",
+    "@types/validator": "^13.11.9",
+    "eslint-config-prettier": "^9.0.0",
+    "eslint-plugin-prettier": "^5.0.0",
     "nodemon": "^3.1.0",
-    "ts-node-dev": "^2.0.0"
+    "prettier": "^3.0.3",
+    "ts-node": "10.9.2"
```

Comment:
Swapped `ts-node-dev` out for `ts-node` since it's being updated, and `nodemon` can be utilized for restarting the server on code-change